staging/src/k8s.io/legacy-cloud-providers/openstack/openstack.go

```diff
@@ -290,11 +290,11 @@ func (os *OpenStack) setConfigFromSecret() error {
 
 		err = gcfg.ReadStringInto(cfg, string(content))
 		if err != nil {
-			return fmt.Errorf("cannot parse data from the secret")
+			return fmt.Errorf("cannot parse data from the secret: %s", err)
```

Comment:
This might result in a secret leak in the logs, when `gcfg` reports the failing source. Please consider reverting this change